### PR TITLE
Enable aio-max-nr as default

### PR DESCRIPTION
**Pull request by RafalKorepta**
The rest of tuners will be adjusted in the next few PRs.

### CODE REVIEW

**Comment by RafalKorepta** [comment]
The test failed on
```
 helm upgrade redpanda-4ig92pvb1x charts/redpanda --namespace redpanda-4ig92pvb1x --wait --timeout 600s
Error: UPGRADE FAILED: timed out waiting for the condition
```
https://github.com/redpanda-data/helm-charts/actions/runs/4222785566/jobs/7331716872#step:10:135

The filesystem turns out is read-only
```
kubectl --request-timeout=30s logs redpanda-4ig92pvb1x-2 --namespace redpanda-4ig92pvb1x --container tuning
  TUNER                  APPLIED  ENABLED  SUPPORTED  ERROR                             
  aio_events             false    true     true       open /proc/sys/fs/aio-max-nr:     
                                                      read-only file system  
```

File: charts/redpanda/templates/statefulset.yaml [hunk 1]
```diff
@@ -53,6 +53,25 @@ spec:
       securityContext: {{ include "pod-security-context" . | nindent 8 }}
       serviceAccountName: {{ include "redpanda.serviceAccountName" . }}
       initContainers:
+{{- if and (hasKey $values.tuning "tune_aio_events") $values.tuning.tune_aio_events }}
+        - name: tuning
```
[the remark this answers is not in the document]
Comment (RafalKorepta) [reply charts/redpanda/templates/statefulset.yaml:57]: Right, I will rename

File: charts/redpanda/values.yaml [hunk 2]
```diff
@@ -549,13 +549,15 @@ rbac:
...
   # This section contains Redpanda tuning parameters.
   # Each parameter below is set to their default values.
   # Remove the curly brackets above if you uncomment any parameters below.
   #
-  # Increases the number of allowed asynchronous IO events.
-  # tune_aio_events: false
+  # Increases the maximum number of outstanding asynchronous IO operations if the
+  # current value is below a certain threshold. This allows redpanda to make as many
+  # simultaneous IO requests as possible, increasing throughput.
+  tune_aio_events: true
```
Comment (joejulian) [inline charts/redpanda/values.yaml:560]: What about adding something like this?

```suggestion
  # Increases the maximum number of outstanding asynchronous IO operations if the
  # current value is below a certain threshold. This allows redpanda to make as many
  # simultaneous IO requests as possible, increasing throughput.
  #
  # Enabling this option will create a privileged container. If your security profile does not allow this,
  # see https://docs.redpanda.com/<blah blah blah> for tuning requirements.
  tune_aio_events: true
```

Comment (joejulian) [reply charts/redpanda/values.yaml:560]: assuming we have a "blah blah blah" page that describes this. :grin:

**Review by joejulian — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

**Pull request merged**